samples/kubernetes/end2end/docs/04-wls-domain.md

```diff
@@ -0,0 +1,119 @@
+## Running a WLS Domain
```

Comment:
Instead of duplicating the entire Operator Quick Start Guide would you not prefer to just add a link to it ?

Reply:
The domain configuration is tailored to fit in this sample. It has specific resources defined, e.g.  a datasource which has parameters to connect to the mysql database created in this sample, a jdbc strore, jms resources. With the resources we can show some concrete data in the dashboard. And it has the exporter web app deployed with the specific configuration.
I don't think it's easy to let user refer to the Operator Quick Start Guide and figure out how to put all the things together.